lightningd/routehint.c

```diff
@@ -106,6 +106,11 @@ routehint_candidates(const tal_t *ctx,
 
 		/* Check channel is in CHANNELD_NORMAL */
 		candidate.c = find_channel_by_scid(peer, &r->short_channel_id);
+
+		/* Try seeing if we should be using a remote alias instead, */
```

Comment:
This confused me, but then I realized this could happen for local channels which don't have a real scid yet: we will put the alias in the "short_channel_id" field of listpeers.  Which is, maybe, a mistake, but it's also reasonable.

Probably deserves a comment here on why listpeers gave us an alias?